src/App.jsx

```diff
@@ -14,57 +15,73 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGood, setSelectedGood] = useState('Jam');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
-      <button data-cy="ClearButton" type="button" className="delete ml-3" />
-    </h1>
+  const addButtonHandler = good => {
+    setSelectedGood(good);
+  };
 
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button data-cy="AddButton" type="button" className="button">
-              +
-            </button>
-          </td>
+  const clearButtonHandler = () => {
+    setSelectedGood('');
+  };
 
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Dumplings
-          </td>
-        </tr>
+  return (
+    <main className="section container">
+      <h1 className="title is-flex is-align-items-center">
+        {selectedGood ? `${selectedGood} is selected` : 'No goods selected'}
+      </h1>
 
-        <tr data-cy="Good" className="has-background-success-light">
-          <td>
-            <button
-              data-cy="RemoveButton"
-              type="button"
-              className="button is-info"
-            >
-              -
-            </button>
-          </td>
-
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Jam
-          </td>
-        </tr>
+      {selectedGood && (
+        <h1 className="title is-flex is-align-items-center">
+          <button
+            onClick={clearButtonHandler}
+            data-cy="ClearButton"
+            type="button"
+            className="delete ml-3"
+          />
+        </h1>
```

Comment:
Instead of using two `<h1/>`s you should render both the button and the title in the same one!

This causes a visual issue:

Your site:
![image](https://github.com/mate-academy/react_goods-selector-js/assets/52002856/3cceff03-6281-4b41-b677-b94811acd332)

The example:
![image](https://github.com/mate-academy/react_goods-selector-js/assets/52002856/2b019c35-5d81-47ba-bd1e-5346b7ad5bef)

See how the clear button is showing up underneath the title in yours and beside the title in the example?

Reply:
Done.